packages/amplify_api/android/gradle/wrapper/gradle-wrapper.properties

```
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.2-all.zip
```

The latest version is 6.6.1.  Any reason not to use it?

```suggestion
distributionUrl=https\://services.gradle.org/distributions/gradle-6.6.1-all.zip
```